src/Exception/FeedFileOperationsException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Feed file operation exception.
+ *
+ * @package Automattic\WooCommerce\Pinterest\Exception
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Pinterest\Exception;
+
+use Exception;
+
+/**
+ * An exception thrown then something went wrong writing into a feed file.
+ */
+class FeedFileOperationsException extends Exception {
```

Comment:
This should also implement the `PinterestException` interface.

```suggestion
class FeedFileOperationsException extends Exception implements PinterestException {
```

Reply:
Done.